recipes/cern-root/all/conanfile.py

```diff
@@ -0,0 +1,315 @@
+                    "builtin_gl2ps": "ON",  # FIXME : replace with gl2ps CCI package when available
+                    "builtin_ftgl": "ON",  # FIXME : replace with ftgl CCI package when available
+                    "builtin_vdt": "ON",  # FIXME : replace with vdt CCI package when available
+                    # No Conan packages available for these dependencies yet
+                    "davix": "OFF",  # FIXME : switch on if davix CCI package available
+                    "pythia6": "OFF",  # FIXME : switch on if pythia6 CCI package available
+                    "pythia8": "OFF",  # FIXME : switch on if pythia8 CCI package available
+                    "mysql": "OFF",  # FIXME : switch on if mysql CCI package available
+                    "oracle": "OFF",
+                    "pgsql": "OFF",  # FIXME: switch on if PostgreSQL CCI package available
+                    "gfal": "OFF",  # FIXME: switch on if gfal CCI package available
+                    "tmva-pymva": "OFF",  # FIXME: switch on if Python CCI package available
+                    "xrootd": "OFF",  # FIXME: switch on if xrootd CCI package available
+                    "pyroot": self._pyrootopt,
+                    # Tell CMake where to look for Conan provided depedencies
+                    "CMAKE_LIBRARY_PATH": cmakelibpath.replace("\\", "/"),
+                    "CMAKE_INCLUDE_PATH": cmakeincludepath.replace("\\", "/"),
+                    # Configure install directories
+                    # Conan CCI hooks restrict the allowed install directory
+                    # names but ROOT is very picky about where build/runtime
+                    # resources get installed.
+                    # Set install prefix to work around these limitations
+                    # Following: https://github.com/conan-io/conan/issues/3695
+                    "CMAKE_INSTALL_PREFIX": os.sep.join((self.package_folder, "res")),
+                    # Fix some Conan-ROOT CMake variable naming differences
+                    "PNG_PNG_INCLUDE_DIR": ";".join(
+                        self.deps_cpp_info["libpng"].include_paths
+                    ).replace("\\", "/"),
+                    "LIBLZMA_INCLUDE_DIR": ";".join(
+                        self.deps_cpp_info["xz_utils"].include_paths
+                    ).replace("\\", "/"),
+                },
+            )
+        return self._cmake
+
+    def _move_findcmake_conan_to_root_dir(self):
+        for f in ["opengl_system", "GLEW", "glu", "TBB", "LibXml2", "ZLIB", "SQLite3"]:
+            shutil.copy(
+                "Find{}.cmake".format(f),
+                os.path.join(self._source_subfolder, "cmake", "modules"),
+            )
+
+    @property
+    def _cmake_cxx_standard(self):
+        compileropt = self.settings.compiler.get_safe("cppstd")
+        if compileropt:
+            return str(compileropt)
+        else:
+            return "11"
```

Comment:
```suggestion
        return self.settings.compiler.get_safe("cppstd", 11)
```

Reply:
I don't think that this suggested change will work as according to the [documentation](https://docs.conan.io/en/latest/reference/conanfile/attributes.html) `get_safe` might return `None`. That case needs to be handled in this method.

Reply:
`self.settings.compiler.get_safe("cppstd", 11)` fallbacks to `11` when cppstd doesn't exist

Reply:
Sorry. I misread your suggestion and didn't see the trailing `,11)`.

This is implemented in 109fb72.